src/parser/CarsParser.java

```diff
@@ -0,0 +1,43 @@
+package parser;
+
+import static parser.AttributeValueParser.*;
+
+import entity.Car;
+import entity.Company;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CarsParser {
+    private static List<Car> cars = new ArrayList<>();
```

Comment:
You don't need to store cars in parser. Role of the parser is one only - parse list of strings.

Reply:
Move it to the method.